lib/dataTableHelpers.tsx

```diff
@@ -22,6 +22,14 @@ export function getDefaultDataTableStyle() {
         cells: {
             style: {
                 fontSize: 14,
+                lineHeight: '1.5em',
+                display: 'block',
+            },
+        },
+        rows: {
+            style: {
+                paddingTop: '10px',
+                paddingBottom: '10px',
```

Comment:
Just a heads up. This will affect every data table we have for the entire web site. Not just the data standards page.

Reply:
ah, did not realize that.